server/src/main/java/org/candlepin/model/CandlepinMode.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2009 - 2012 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.model;
+
+import java.io.Serializable;
+import java.util.Date;
+
+/**
+ * Class representing candlepins mode, (NORMAL || SUSPEND),
+ * and the associated reason for this mode.
+ */
+public class CandlepinMode implements Serializable {
+    private static final long serialVersionUID = -7059065874812188168L;
+
+    private Mode mode;
```

Comment:
ack, renamed CandlepinMode -> CandlepinModeChange, so now it is CandlepinModeChange and Mode, which should be clear enough.